README.md

```diff
@@ -1 +1,53 @@
-# To-Do-List
\ No newline at end of file
+![Microverse](https://img.shields.io/badge/Microverse-blueviolet) ![HTML](https://img.shields.io/badge/-HTML-orange) ![CSS](https://img.shields.io/badge/-CSS-blue) ![JavaScript](https://img.shields.io/badge/-JavaScript-yellow)
```

Comment:
- [ ] The app is not working so I can't check the local storage or the functionality of the app, kindly fix it so the app is working properly
- [ ] [OPTIONAL] Kindly follow the app design as per the requirement guidelines 

![Screenshot_20220820_061810](https://user-images.githubusercontent.com/25227022/185730321-f866c1bc-9abd-41be-bdf5-5df27952e85c.png)